docs/developers/scripts/windows_postinstall.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Install NSIS on test machines, any other post-install
+# This should be run with administrative privileges.
+
+set -eu -o pipefail
+set -x
+
+NSIS_VERSION=3.06.1
+NSIS_HOME="/c/Program Files (x86)/NSIS"
+
+# Note that this is not a silent NSIS install because the silent install
+# does not install the default Plugins and Stubs
+curl -sSL -o /tmp/nsis-setup.exe https://prdownloads.sourceforge.net/nsis/nsis-${NSIS_VERSION}-setup.exe?download && chmod +x /tmp/nsis-setup.exe
+/tmp/nsis-setup.exe
+
+# Get the Plugins for NSIS
+curl -fsSL -o /tmp/EnVar-Plugin.zip https://github.com/GsNSIS/EnVar/releases/latest/download/EnVar-Plugin.zip && unzip -o -d "${NSIS_HOME}" /tmp/EnVar-Plugin.zip
+curl -fsSL -o /tmp/INetC.zip https://github.com/DigitalMediaServer/NSIS-INetC-plugin/releases/latest/download/INetC.zip && unzip -o -d "${NSIS_HOME}/Plugins" /tmp/INetC.zip
+
+echo "You must now add to the system path C:\Program Files (x86)\NSIS\bin, which for some reason is not added by the installer"
```

Comment:
Maybe you like to setup the path automatically, not tested but should work
```suggestion
setx Path "%Path%;%ProgramFiles(x86)\NSIS\bin" /M
```

Reply:
The `setx` with `%Path%` can't work here right, since this is a bash script? And of course there's the problem of getting it in there too many times. 

I also saw *SO MANY WARNINGS* in https://stackoverflow.com/questions/9546324/adding-a-directory-to-the-path-environment-variable-in-windows that I didn't even try.

Have no idea why the installer doesn't add to the path, or at least offer the option.